Neos.Neos/Classes/Command/UserCommandController.php

```diff
@@ -397,6 +425,14 @@ protected function getTableRowForUser(User $user)
                 $roleNames[] = $role->getIdentifier();
             }
         }
-        return [$user->getName()->getFullName(), $user->getPrimaryElectronicAddress(), implode(', ', $accountIdentifiers), implode(', ', $roleNames), ($user->isActive() ? 'yes' : 'no')];
+        return [
+            $user->getName()->getFullName(),
+            $user->getPrimaryElectronicAddress() ?? 'No Primary Electronic Address set',
+            implode(', ', $accountIdentifiers),
+            implode(', ', $roleNames),
+            ($user->isActive() ? 'yes' : 'no'),
+            $account->getCreationDate()->format('Y-m-d'),
```

Comment:
You don't define $account anywhere.

Reply:
Would it be enough to define it like that for example?:

```php
private Account $account
```

Reply:
As a user can have multiple accounts you need to decide for which one you want to output the date